packages/fast-form-generator-react/src/utilities/form-input.style.ts

```diff
@@ -270,7 +270,7 @@ export function applyPopupMenuStyles(): ICSSRules<{}> {
         width: toPx(200),
         background: colors.menuGray,
         zIndex: "1",
-        top: toPx(55),
+        top: toPx(78),
```

Comment:
this looks wrong to me.
![screen shot 2018-07-11 at 4 40 30 pm](https://user-images.githubusercontent.com/7559015/42604748-31a1969e-8529-11e8-80f1-de158163ff04.png)

Reply:
<img width="380" alt="screen shot 2018-07-11 at 7 23 03 pm" src="https://user-images.githubusercontent.com/22480660/42609137-e0f4e388-853f-11e8-94d0-886f3c081c15.png">
Looks fine to me? Where are you seeing this?